docs/Code Exposure.md

```diff
@@ -0,0 +1,261 @@
+- DefenseEvasion:IAMUser/AnomalousBehavior
+- Discovery:IAMUser/AnomalousBehavior
+- Exfiltration:IAMUser/AnomalousBehavior
+- Exfiltration:S3/MaliciousIPCaller
+- Exfiltration:S3/ObjectRead.Unusual
+- Impact:IAMUser/AnomalousBehavior
+- InitialAccess:IAMUser/AnomalousBehavior
+- Persistence:IAMUser/AnomalousBehavior
+- PrivilegeEscalation:IAMUser/AnomalousBehavior
+- UnauthorizedAccess:IAMUser/InstanceCredentialExfiltration.InsideAWS
+- UnauthorizedAccess:IAMUser/InstanceCredentialExfiltration.OutsideAWS
+
+### Objectives
+Throughout the execution of the playbook, focus on the _***desired outcomes***_, taking notes for enhancement of incident response capabilities.
+
+#### Determine: 
+* **Code copies, transfers, and publication**
+* **Credential exposure**
+* **Vulnerabilities exposed**
+* **Enviormental intelligence exposed**
+	* **Tools used**
+	* **Configuration information**
+* **Actor's intent**
+* **Actor's attribution**
+* **Other damage inflicted to the environment and business**
+* **Risk created for environment and business**
+
+#### Recover:
+* **Expire or reset any exposed authentication credentials**
+* **Enumerate risks for potential attack vectors based on exposed environmental intelligence**
+* **Minimize or eliminate risks of exposed code**
+
+#### Enhance CAF Security Perspective components:
+[AWS Cloud Adoption Framework Security Perspective](https://d0.awsstatic.com/whitepapers/AWS_CAF_Security_Perspective.pdf)
+* **Directive**
+* **Detective**
+* **Responsive**
+* **Preventative**
+
+![Image](/images/aws_caf.png)
+* * *
+
+### Response Steps
+1. [**PREPARATION**] Create an account asset inventory
+2. [**PREPARATION**] Create a repository inventory
+3. [**PREPARATION**] Enable logging as appropriate
+4. [**PREPARATION**] Identify what type of data is in each repository
+5. [**PREPARATION**] Identify, document, and test escalation procedures
+6. [**PREPARATION**] Implement training to address exfiltration attacks
+7. [**DETECTION AND ANALYSIS**] Perform exfiltration and DLP checks
+8. [**DETECTION AND ANALYSIS**] Review respository (CodeCommit) read and write actions
+9. [**DETECTION AND ANALYSIS**] Review DNS logs
+10. [**DETECTION AND ANALYSIS**] Review VPC flow logs
+11. [**DETECTION AND ANALYSIS**] Review endpoint / host based logs
+12. [**CONTAINMENT**] Block access for affected accounts
+13. [**ERADICATION**] Remove unrecognized and unauthorized objects in respositories
+14. [**RECOVERY**] Perform recovery procedures as appropriate
+
+***The response steps follow the Incident Response Life Cycle from [NIST Special Publication 800-61r2 Computer Security Incident Handling Guide](https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-61r2.pdf)  
+
+![Image](/images/nist_life_cycle.png)***
+
+### Incident Classification & Handling
+* **Tactics, techniques, and procedures**: Data Exfiltration, AWS Service Abuse
+* **Category**: Data Loss
+* **Resource**: CodeCommit, S3, EC2
```

Comment:
I see CodeCommit mentioned here. Do we have incident response or review procedures for CodeCommit? 

Document focuses on S3, but I didn't see anything related to reviewing CodeCommit events beyond asset inventory

Reply:
Added content for CodeCommit and EC2.